docs/configuration/integrations/integrations-next/eventhandler-config.md

```diff
@@ -0,0 +1,300 @@
+spec:
+  ports:
+  - port: 12345
+    name: http-metrics
+  clusterIP: None
+  selector:
+    name: grafana-agent-eventhandler
+---
+kind: ConfigMap
+metadata:
+  name: grafana-agent-eventhandler
+  namespace: default
+apiVersion: v1
+data:
+  agent.yaml: |
+    server:
+      http_listen_port: 12345
+      log_level: info
+  
+    integrations:
+      eventhandler:
+        cluster_name: "cloud"
+        cache_path: "/etc/eventhandler/eventhandler.cache"
+      
+    logs:
+      configs:
+      - name: default
+        clients:
+        - url: https://logs-prod-us-central1.grafana.net/api/prom/push
+          basic_auth:
+            username: YOUR_LOKI_USER
+            password: YOUR_LOKI_API_KEY
+        positions:
+          filename: /tmp/positions0.yaml
+      - name: eventhandler_logs
+        clients:
+        - url: https://logs-prod-us-central1.grafana.net/api/prom/push
+          basic_auth:
+            username: YOUR_LOKI_USER
+            password: YOUR_LOKI_API_KEY
+        scrape_configs:
+        - job_name: eventhandler_logs
+          kubernetes_sd_configs:
+            - role: pod
+          pipeline_stages:
+            - docker: {}
+          relabel_configs:
+            - source_labels:
+                - __meta_kubernetes_pod_node_name
+              target_label: __host__
+            - action: keep
+              regex: grafana-agent-eventhandler-*
+              source_labels:
+                - __meta_kubernetes_pod_name
+            - action: labelmap
+              regex: __meta_kubernetes_pod_label_(.+)
+            - action: replace
+              replacement: $1
+              source_labels:
+                - __meta_kubernetes_pod_name
+              target_label: job
+            - action: replace
+              source_labels:
+                - __meta_kubernetes_namespace
+              target_label: namespace
+            - action: replace
+              source_labels:
+                - __meta_kubernetes_pod_name
+              target_label: pod
+            - action: replace
+              source_labels:
+                - __meta_kubernetes_pod_container_name
+              target_label: container
+            - replacement: /var/log/pods/*$1/*.log
+              separator: /
+              source_labels:
+                - __meta_kubernetes_pod_uid
+                - __meta_kubernetes_pod_container_name
+              target_label: __path__
+        positions:
+          filename: /tmp/positions1.yaml
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: grafana-agent-eventhandler
+  namespace: default
+spec:
+  serviceName: "grafana-agent-eventhandler-svc"
+  selector:
+    matchLabels:
+      name: grafana-agent-eventhandler
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: grafana-agent-eventhandler
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+      - name: agent
+        image: grafana/agent:dev.eventlogger-781ee75
```

Comment:
```suggestion
        image: grafana/agent:main
```

This probably doesn't matter much but once it's merged in, users can pull the main image.